### PR TITLE
Release v0.28.0

### PR DESCRIPTION
**Pull request by jchengsfx**
Requires #3745

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,65 @@
...
+  - Add support for summary datatype (#3660)
+  - Add datadog span operation name remapping config option (#3444)
+  - Update error formatting for error spans that are not exceptions (#3701)
+- `nginx` receiver: Update the nginx metrics to more closely align with the conventions (#3420)
+- `elasticsearch` exporter: Init JSON encoding support (#3101)
+- `jmx` receiver:
+  - Allow setting system properties (#3450)
+  - Update tested JMX Metric Gatherer release (#3695)
+- Refactor components for the Client Authentication Extensions (#3507)
+- Remove usage of `tracetranslator` bigendian helpers in `jaegerthrifthttp` exporter and `tailsampling` processor (#3451)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:40]: This is an internal change, not interesting for user, let's remove from the changelog.

**Review by tigrannajaryan — APPROVED** [review]
LGTM, except one extra line.

File: CHANGELOG.md (hunk 1 above)
Comment (jchengsfx) [reply CHANGELOG.md]: Thanks, line removed

**Comment by sodabrew** [comment]
Could you create the GitHub Release entry for this at https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.28.0 ?

**Pull request merged**